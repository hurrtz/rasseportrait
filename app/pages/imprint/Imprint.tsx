import { Container, Text, Title, Card } from "@mantine/core";

const Imprint = () => {
  return (
    <Container size="md">
      <Title order={1}>Impressum</Title>
      <Card withBorder p="lg" mt="lg">
        <Title order={2}>Zweck</Title>
        <Text mt="lg">
          Diese Website ist ein Fanprojekt zum Podcast{" "}
          <a href="https://plus.rtl.de/podcast/tierisch-menschlich-der-podcast-mit-hundeprofi-martin-ruetter-und-katharina-adick-m5iuweomug8fv">
            Tierisch Menschlich
          </a>{" "}
          von Martin Rütter und Katharina Adick. Es enthält eine nicht
          verbindliche kuratierte Liste aller Episoden, die ein Rasseportrait
          enthalten. Dieses Projekt ist in keiner Weise beauftragt von RTL oder
          einer der teilnehmenden oder der Produktion des Podcasts mitwirkenden
          Personen.
        </Text>
        <Text>
          Bei Problemen, Änderungswünschen oder sonstigem Feedback wenden Sie
          sich gern an die unten stehende Kontaktmöglichkeit.
        </Text>
      </Card>
      <Card withBorder p="lg" mt="lg">
        <Title order={2}>Kontakt</Title>
        <Text mt="lg">
          <strong>Name:</strong> Tobias Winkler
          <br />
          <strong>E-Mail:</strong>{" "}
          <a href="mailto:rasseportrait@tobiaswinkler.berlin">
            rasseportrait@tobiaswinkler.berlin
          </a>
          <br />
          <strong>LinkedIn:</strong>{" "}
          <a
            href="https://www.linkedin.com/in/tobias-winkler-87a08210b/"
            target="_blank"
          >
            tobias-winkler-87a08210b
          </a>
          <br />
          <strong>GitHub:</strong>{" "}
          <a href="https://github.com/hurrtz" target="_blank">
            hurrtz
          </a>
        </Text>
      </Card>
    </Container>
  );
};

export default Imprint;
