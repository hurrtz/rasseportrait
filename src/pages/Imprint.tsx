import React from "react";
import Typography from "@mui/material/Typography";
import Box from "@mui/material/Box";
import List from "@mui/material/List";
import ListItem from "@mui/material/ListItem";
import ListItemButton from "@mui/material/ListItemButton";
import ListItemText from "@mui/material/ListItemText";
import ListItemAvatar from "@mui/material/ListItemAvatar";
import Avatar from "@mui/material/Avatar";
import EmailIcon from "@mui/icons-material/Email";
import BadgeIcon from "@mui/icons-material/Badge";
import LinkedInIcon from "@mui/icons-material/LinkedIn";
import GitHubIcon from "@mui/icons-material/GitHub";
import Card from "@mui/material/Card";
import CardContent from "@mui/material/CardContent";

const handleMailClick = () => {
  window.open("mailto:rasseportrait@tobiaswinkler.berlin", "_blank");
};

const handleURLClick = (url: string) => {
  window.open(url, "_blank");
};

const Imprint = () => (
  <Box>
    <Typography variant="h2" gutterBottom>
      Impressum
    </Typography>

    <Card>
      <CardContent>
        <Typography variant="h3" gutterBottom>
          Zweck
        </Typography>

        <Typography variant="body2" gutterBottom>
          Diese Website ist ein Fanprojekt zum Podcast{" "}
          <a
            href="https://plus.rtl.de/podcast/tierisch-menschlich-der-podcast-mit-hundeprofi-martin-ruetter-und-katharina-adick-m5iuweomug8fv"
            target="_blank"
          >
            Tierisch Menschlich
          </a>{" "}
          von Martin Rütter und Katharina Adick. Es enthält eine nicht
          verbindliche kuratierte Liste aller Episoden, die ein Rasseportrait
          enthalten. Dieses Projekt ist in keiner Weise beauftragt von RTL oder
          einer der teilnehmenden oder der Produktion des Podcasts mitwirkenden
          Personen.
        </Typography>

        <Typography variant="body2" gutterBottom mt={2}>
          Bei Problemen, Änderungswünschen oder sonstigem Feedback wenden Sie
          sich gern an die unten stehende Kontaktmöglichkeit.
        </Typography>
      </CardContent>
    </Card>

    <Card sx={{ marginTop: 4 }}>
      <CardContent>
        <Typography variant="h3" gutterBottom>
          Kontakt
        </Typography>

        <List>
          <ListItem>
            <ListItemAvatar>
              <Avatar>
                <BadgeIcon />
              </Avatar>
            </ListItemAvatar>
            <ListItemText primary="Tobias Winkler" />
          </ListItem>
          <ListItem disablePadding>
            <ListItemButton>
              <ListItemAvatar>
                <Avatar>
                  <EmailIcon />
                </Avatar>
              </ListItemAvatar>
              <ListItemText
                primary="rasseportrait@tobiaswinkler.berlin"
                onClick={handleMailClick}
              />
            </ListItemButton>
          </ListItem>
          <ListItem disablePadding>
            <ListItemButton>
              <ListItemAvatar>
                <Avatar>
                  <LinkedInIcon />
                </Avatar>
              </ListItemAvatar>
              <ListItemText
                primary="LinkedIn"
                onClick={() =>
                  handleURLClick(
                    "https://www.linkedin.com/in/tobias-winkler-87a08210b/",
                  )
                }
              />
            </ListItemButton>
          </ListItem>
          <ListItem disablePadding>
            <ListItemButton>
              <ListItemAvatar>
                <Avatar>
                  <GitHubIcon />
                </Avatar>
              </ListItemAvatar>
              <ListItemText
                primary="Github"
                onClick={() => handleURLClick("https://github.com/hurrtz")}
              />
            </ListItemButton>
          </ListItem>
        </List>
      </CardContent>
    </Card>
  </Box>
);

export default Imprint;
