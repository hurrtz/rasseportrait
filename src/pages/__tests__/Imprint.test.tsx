import React from "react";
import renderer from "react-test-renderer";
import Imprint from "../Imprint";

it("renders correctly", () => {
  const component = renderer.create(<Imprint />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
