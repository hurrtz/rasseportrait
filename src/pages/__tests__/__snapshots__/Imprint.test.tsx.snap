// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiBox-root css-0"
>
  <style
    dangerouslySetInnerHTML={
      {
        "__html": ".css-i1hrn3-MuiTypography-root{margin:0;font-family:"Roboto","Helvetica","Arial",sans-serif;font-weight:300;font-size:3.75rem;line-height:1.2;letter-spacing:-0.00833em;margin-bottom:0.35em;}",
      }
    }
    data-emotion="css i1hrn3-MuiTypography-root"
  />
  <h2
    className="MuiTypography-root MuiTypography-h2 MuiTypography-gutterBottom css-i1hrn3-MuiTypography-root"
  >
    Impressum
  </h2>
  <style
    dangerouslySetInnerHTML={
      {
        "__html": ".css-1gdfwye-MuiCard-root{overflow:hidden;}",
      }
    }
    data-emotion="css 1gdfwye-MuiCard-root"
  />
  <style
    dangerouslySetInnerHTML={
      {
        "__html": ".css-bhp9pd-MuiPaper-root-MuiCard-root{background-color:#fff;color:rgba(0, 0, 0, 0.87);-webkit-transition:box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;border-radius:4px;box-shadow:0px 2px 1px -1px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 1px 3px 0px rgba(0,0,0,0.12);overflow:hidden;}",
      }
    }
    data-emotion="css bhp9pd-MuiPaper-root-MuiCard-root"
  />
  <div
    className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-bhp9pd-MuiPaper-root-MuiCard-root"
  >
    <style
      dangerouslySetInnerHTML={
        {
          "__html": ".css-46bh2p-MuiCardContent-root{padding:16px;}.css-46bh2p-MuiCardContent-root:last-child{padding-bottom:24px;}",
        }
      }
      data-emotion="css 46bh2p-MuiCardContent-root"
    />
    <div
      className="MuiCardContent-root css-46bh2p-MuiCardContent-root"
    >
      <style
        dangerouslySetInnerHTML={
          {
            "__html": ".css-1mktp8m-MuiTypography-root{margin:0;font-family:"Roboto","Helvetica","Arial",sans-serif;font-weight:400;font-size:3rem;line-height:1.167;letter-spacing:0em;margin-bottom:0.35em;}",
          }
        }
        data-emotion="css 1mktp8m-MuiTypography-root"
      />
      <h3
        className="MuiTypography-root MuiTypography-h3 MuiTypography-gutterBottom css-1mktp8m-MuiTypography-root"
      >
        Zweck
      </h3>
      <style
        dangerouslySetInnerHTML={
          {
            "__html": ".css-ejgmvm-MuiTypography-root{margin:0;font-family:"Roboto","Helvetica","Arial",sans-serif;font-weight:400;font-size:0.875rem;line-height:1.43;letter-spacing:0.01071em;margin-bottom:0.35em;}",
          }
        }
        data-emotion="css ejgmvm-MuiTypography-root"
      />
      <p
        className="MuiTypography-root MuiTypography-body2 MuiTypography-gutterBottom css-ejgmvm-MuiTypography-root"
      >
        Diese Website ist ein Fanprojekt zum Podcast
         
        <a
          href="https://plus.rtl.de/podcast/tierisch-menschlich-der-podcast-mit-hundeprofi-martin-ruetter-und-katharina-adick-m5iuweomug8fv"
          target="_blank"
        >
          Tierisch Menschlich
        </a>
         
        von Martin Rütter und Katharina Adick. Es enthält eine nicht verbindliche kuratierte Liste aller Episoden, die ein Rasseportrait enthalten. Dieses Projekt ist in keiner Weise beauftragt von RTL oder einer der teilnehmenden oder der Produktion des Podcasts mitwirkenden Personen.
      </p>
      <style
        dangerouslySetInnerHTML={
          {
            "__html": ".css-1ewbluu-MuiTypography-root{margin:0;font-family:"Roboto","Helvetica","Arial",sans-serif;font-weight:400;font-size:0.875rem;line-height:1.43;letter-spacing:0.01071em;margin-bottom:0.35em;margin-top:16px;}",
          }
        }
        data-emotion="css 1ewbluu-MuiTypography-root"
      />
      <p
        className="MuiTypography-root MuiTypography-body2 MuiTypography-gutterBottom css-1ewbluu-MuiTypography-root"
      >
        Bei Problemen, Änderungswünschen oder sonstigem Feedback wenden Sie sich gern an die unten stehende Kontaktmöglichkeit.
      </p>
    </div>
  </div>
  <style
    dangerouslySetInnerHTML={
      {
        "__html": ".css-tj3e9a-MuiCard-root{overflow:hidden;margin-top:32px;}",
      }
    }
    data-emotion="css tj3e9a-MuiCard-root"
  />
  <style
    dangerouslySetInnerHTML={
      {
        "__html": ".css-w2s3lu-MuiPaper-root-MuiCard-root{background-color:#fff;color:rgba(0, 0, 0, 0.87);-webkit-transition:box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;border-radius:4px;box-shadow:0px 2px 1px -1px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 1px 3px 0px rgba(0,0,0,0.12);overflow:hidden;margin-top:32px;}",
      }
    }
    data-emotion="css w2s3lu-MuiPaper-root-MuiCard-root"
  />
  <div
    className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-w2s3lu-MuiPaper-root-MuiCard-root"
  >
    <div
      className="MuiCardContent-root css-46bh2p-MuiCardContent-root"
    >
      <h3
        className="MuiTypography-root MuiTypography-h3 MuiTypography-gutterBottom css-1mktp8m-MuiTypography-root"
      >
        Kontakt
      </h3>
      <style
        dangerouslySetInnerHTML={
          {
            "__html": ".css-h4y409-MuiList-root{list-style:none;margin:0;padding:0;position:relative;padding-top:8px;padding-bottom:8px;}",
          }
        }
        data-emotion="css h4y409-MuiList-root"
      />
      <ul
        className="MuiList-root MuiList-padding css-h4y409-MuiList-root"
      >
        <style
          dangerouslySetInnerHTML={
            {
              "__html": ".css-1p823my-MuiListItem-root{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-box-pack:start;-ms-flex-pack:start;-webkit-justify-content:flex-start;justify-content:flex-start;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;position:relative;-webkit-text-decoration:none;text-decoration:none;width:100%;box-sizing:border-box;text-align:left;padding-top:8px;padding-bottom:8px;padding-left:16px;padding-right:16px;}.css-1p823my-MuiListItem-root.Mui-focusVisible{background-color:rgba(0, 0, 0, 0.12);}.css-1p823my-MuiListItem-root.Mui-selected{background-color:rgba(25, 118, 210, 0.08);}.css-1p823my-MuiListItem-root.Mui-selected.Mui-focusVisible{background-color:rgba(25, 118, 210, 0.2);}.css-1p823my-MuiListItem-root.Mui-disabled{opacity:0.38;}",
            }
          }
          data-emotion="css 1p823my-MuiListItem-root"
        />
        <li
          className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-1p823my-MuiListItem-root"
          disabled={false}
        >
          <style
            dangerouslySetInnerHTML={
              {
                "__html": ".css-1e9lk82-MuiListItemAvatar-root{min-width:56px;-webkit-flex-shrink:0;-ms-flex-negative:0;flex-shrink:0;}",
              }
            }
            data-emotion="css 1e9lk82-MuiListItemAvatar-root"
          />
          <div
            className="MuiListItemAvatar-root css-1e9lk82-MuiListItemAvatar-root"
          >
            <style
              dangerouslySetInnerHTML={
                {
                  "__html": ".css-17o22dy-MuiAvatar-root{position:relative;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;-webkit-box-pack:center;-ms-flex-pack:center;-webkit-justify-content:center;justify-content:center;-webkit-flex-shrink:0;-ms-flex-negative:0;flex-shrink:0;width:40px;height:40px;font-family:"Roboto","Helvetica","Arial",sans-serif;font-size:1.25rem;line-height:1;border-radius:50%;overflow:hidden;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;color:#fff;background-color:#bdbdbd;}",
                }
              }
              data-emotion="css 17o22dy-MuiAvatar-root"
            />
            <div
              className="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-17o22dy-MuiAvatar-root"
            >
              <style
                dangerouslySetInnerHTML={
                  {
                    "__html": ".css-i4bv87-MuiSvgIcon-root{-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;width:1em;height:1em;display:inline-block;fill:currentColor;-webkit-flex-shrink:0;-ms-flex-negative:0;flex-shrink:0;-webkit-transition:fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;font-size:1.5rem;}",
                  }
                }
                data-emotion="css i4bv87-MuiSvgIcon-root"
              />
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="BadgeIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M20 7h-5V4c0-1.1-.9-2-2-2h-2c-1.1 0-2 .9-2 2v3H4c-1.1 0-2 .9-2 2v11c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V9c0-1.1-.9-2-2-2M9 12c.83 0 1.5.67 1.5 1.5S9.83 15 9 15s-1.5-.67-1.5-1.5S8.17 12 9 12m3 6H6v-.75c0-1 2-1.5 3-1.5s3 .5 3 1.5zm1-9h-2V4h2zm5 7.5h-4V15h4zm0-3h-4V12h4z"
                />
              </svg>
            </div>
          </div>
          <style
            dangerouslySetInnerHTML={
              {
                "__html": ".css-tlelie-MuiListItemText-root{-webkit-flex:1 1 auto;-ms-flex:1 1 auto;flex:1 1 auto;min-width:0;margin-top:4px;margin-bottom:4px;}",
              }
            }
            data-emotion="css tlelie-MuiListItemText-root"
          />
          <div
            className="MuiListItemText-root css-tlelie-MuiListItemText-root"
          >
            <style
              dangerouslySetInnerHTML={
                {
                  "__html": ".css-10hburv-MuiTypography-root{margin:0;font-family:"Roboto","Helvetica","Arial",sans-serif;font-weight:400;font-size:1rem;line-height:1.5;letter-spacing:0.00938em;display:block;}",
                }
              }
              data-emotion="css 10hburv-MuiTypography-root"
            />
            <span
              className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
            >
              Tobias Winkler
            </span>
          </div>
        </li>
        <style
          dangerouslySetInnerHTML={
            {
              "__html": ".css-1nxmd3h-MuiListItem-root{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-box-pack:start;-ms-flex-pack:start;-webkit-justify-content:flex-start;justify-content:flex-start;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;position:relative;-webkit-text-decoration:none;text-decoration:none;width:100%;box-sizing:border-box;text-align:left;}.css-1nxmd3h-MuiListItem-root.Mui-focusVisible{background-color:rgba(0, 0, 0, 0.12);}.css-1nxmd3h-MuiListItem-root.Mui-selected{background-color:rgba(25, 118, 210, 0.08);}.css-1nxmd3h-MuiListItem-root.Mui-selected.Mui-focusVisible{background-color:rgba(25, 118, 210, 0.2);}.css-1nxmd3h-MuiListItem-root.Mui-disabled{opacity:0.38;}",
            }
          }
          data-emotion="css 1nxmd3h-MuiListItem-root"
        />
        <li
          className="MuiListItem-root MuiListItem-gutters css-1nxmd3h-MuiListItem-root"
          disabled={false}
        >
          <style
            dangerouslySetInnerHTML={
              {
                "__html": ".css-z4ly38-MuiListItemButton-root{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-box-flex:1;-webkit-flex-grow:1;-ms-flex-positive:1;flex-grow:1;-webkit-box-pack:start;-ms-flex-pack:start;-webkit-justify-content:flex-start;justify-content:flex-start;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;position:relative;-webkit-text-decoration:none;text-decoration:none;min-width:0;box-sizing:border-box;text-align:left;padding-top:8px;padding-bottom:8px;-webkit-transition:background-color 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:background-color 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;padding-left:16px;padding-right:16px;}.css-z4ly38-MuiListItemButton-root:hover{-webkit-text-decoration:none;text-decoration:none;background-color:rgba(0, 0, 0, 0.04);}@media (hover: none){.css-z4ly38-MuiListItemButton-root:hover{background-color:transparent;}}.css-z4ly38-MuiListItemButton-root.Mui-selected{background-color:rgba(25, 118, 210, 0.08);}.css-z4ly38-MuiListItemButton-root.Mui-selected.Mui-focusVisible{background-color:rgba(25, 118, 210, 0.2);}.css-z4ly38-MuiListItemButton-root.Mui-selected:hover{background-color:rgba(25, 118, 210, 0.12);}@media (hover: none){.css-z4ly38-MuiListItemButton-root.Mui-selected:hover{background-color:rgba(25, 118, 210, 0.08);}}.css-z4ly38-MuiListItemButton-root.Mui-focusVisible{background-color:rgba(0, 0, 0, 0.12);}.css-z4ly38-MuiListItemButton-root.Mui-disabled{opacity:0.38;}",
              }
            }
            data-emotion="css z4ly38-MuiListItemButton-root"
          />
          <style
            dangerouslySetInnerHTML={
              {
                "__html": ".css-16ac5r2-MuiButtonBase-root-MuiListItemButton-root{display:-webkit-inline-box;display:-webkit-inline-flex;display:-ms-inline-flexbox;display:inline-flex;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;-webkit-box-pack:center;-ms-flex-pack:center;-webkit-justify-content:center;justify-content:center;position:relative;box-sizing:border-box;-webkit-tap-highlight-color:transparent;background-color:transparent;outline:0;border:0;margin:0;border-radius:0;padding:0;cursor:pointer;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;vertical-align:middle;-moz-appearance:none;-webkit-appearance:none;-webkit-text-decoration:none;text-decoration:none;color:inherit;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-box-flex:1;-webkit-flex-grow:1;-ms-flex-positive:1;flex-grow:1;-webkit-box-pack:start;-ms-flex-pack:start;-webkit-justify-content:flex-start;justify-content:flex-start;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;position:relative;-webkit-text-decoration:none;text-decoration:none;min-width:0;box-sizing:border-box;text-align:left;padding-top:8px;padding-bottom:8px;-webkit-transition:background-color 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:background-color 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;padding-left:16px;padding-right:16px;}.css-16ac5r2-MuiButtonBase-root-MuiListItemButton-root::-moz-focus-inner{border-style:none;}.css-16ac5r2-MuiButtonBase-root-MuiListItemButton-root.Mui-disabled{pointer-events:none;cursor:default;}@media print{.css-16ac5r2-MuiButtonBase-root-MuiListItemButton-root{-webkit-print-color-adjust:exact;color-adjust:exact;}}.css-16ac5r2-MuiButtonBase-root-MuiListItemButton-root:hover{-webkit-text-decoration:none;text-decoration:none;background-color:rgba(0, 0, 0, 0.04);}@media (hover: none){.css-16ac5r2-MuiButtonBase-root-MuiListItemButton-root:hover{background-color:transparent;}}.css-16ac5r2-MuiButtonBase-root-MuiListItemButton-root.Mui-selected{background-color:rgba(25, 118, 210, 0.08);}.css-16ac5r2-MuiButtonBase-root-MuiListItemButton-root.Mui-selected.Mui-focusVisible{background-color:rgba(25, 118, 210, 0.2);}.css-16ac5r2-MuiButtonBase-root-MuiListItemButton-root.Mui-selected:hover{background-color:rgba(25, 118, 210, 0.12);}@media (hover: none){.css-16ac5r2-MuiButtonBase-root-MuiListItemButton-root.Mui-selected:hover{background-color:rgba(25, 118, 210, 0.08);}}.css-16ac5r2-MuiButtonBase-root-MuiListItemButton-root.Mui-focusVisible{background-color:rgba(0, 0, 0, 0.12);}.css-16ac5r2-MuiButtonBase-root-MuiListItemButton-root.Mui-disabled{opacity:0.38;}",
              }
            }
            data-emotion="css 16ac5r2-MuiButtonBase-root-MuiListItemButton-root"
          />
          <div
            className="MuiButtonBase-root MuiListItemButton-root MuiListItemButton-gutters MuiListItemButton-root MuiListItemButton-gutters css-16ac5r2-MuiButtonBase-root-MuiListItemButton-root"
            onBlur={[Function]}
            onContextMenu={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiListItemAvatar-root css-1e9lk82-MuiListItemAvatar-root"
            >
              <div
                className="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-17o22dy-MuiAvatar-root"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="EmailIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M20 4H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2m0 4-8 5-8-5V6l8 5 8-5z"
                  />
                </svg>
              </div>
            </div>
            <div
              className="MuiListItemText-root css-tlelie-MuiListItemText-root"
              onClick={[Function]}
            >
              <span
                className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
              >
                rasseportrait@tobiaswinkler.berlin
              </span>
            </div>
          </div>
        </li>
        <li
          className="MuiListItem-root MuiListItem-gutters css-1nxmd3h-MuiListItem-root"
          disabled={false}
        >
          <div
            className="MuiButtonBase-root MuiListItemButton-root MuiListItemButton-gutters MuiListItemButton-root MuiListItemButton-gutters css-16ac5r2-MuiButtonBase-root-MuiListItemButton-root"
            onBlur={[Function]}
            onContextMenu={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiListItemAvatar-root css-1e9lk82-MuiListItemAvatar-root"
            >
              <div
                className="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-17o22dy-MuiAvatar-root"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="LinkedInIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 3a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h14m-.5 15.5v-5.3a3.26 3.26 0 0 0-3.26-3.26c-.85 0-1.84.52-2.32 1.3v-1.11h-2.79v8.37h2.79v-4.93c0-.77.62-1.4 1.39-1.4a1.4 1.4 0 0 1 1.4 1.4v4.93h2.79M6.88 8.56a1.68 1.68 0 0 0 1.68-1.68c0-.93-.75-1.69-1.68-1.69a1.69 1.69 0 0 0-1.69 1.69c0 .93.76 1.68 1.69 1.68m1.39 9.94v-8.37H5.5v8.37h2.77z"
                  />
                </svg>
              </div>
            </div>
            <div
              className="MuiListItemText-root css-tlelie-MuiListItemText-root"
              onClick={[Function]}
            >
              <span
                className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
              >
                LinkedIn
              </span>
            </div>
          </div>
        </li>
        <li
          className="MuiListItem-root MuiListItem-gutters css-1nxmd3h-MuiListItem-root"
          disabled={false}
        >
          <div
            className="MuiButtonBase-root MuiListItemButton-root MuiListItemButton-gutters MuiListItemButton-root MuiListItemButton-gutters css-16ac5r2-MuiButtonBase-root-MuiListItemButton-root"
            onBlur={[Function]}
            onContextMenu={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiListItemAvatar-root css-1e9lk82-MuiListItemAvatar-root"
            >
              <div
                className="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-17o22dy-MuiAvatar-root"
              >
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="GitHubIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M12 1.27a11 11 0 00-3.48 21.46c.55.09.73-.28.73-.55v-1.84c-3.03.64-3.67-1.46-3.67-1.46-.55-1.29-1.28-1.65-1.28-1.65-.92-.65.1-.65.1-.65 1.1 0 1.73 1.1 1.73 1.1.92 1.65 2.57 1.2 3.21.92a2 2 0 01.64-1.47c-2.47-.27-5.04-1.19-5.04-5.5 0-1.1.46-2.1 1.2-2.84a3.76 3.76 0 010-2.93s.91-.28 3.11 1.1c1.8-.49 3.7-.49 5.5 0 2.1-1.38 3.02-1.1 3.02-1.1a3.76 3.76 0 010 2.93c.83.74 1.2 1.74 1.2 2.94 0 4.21-2.57 5.13-5.04 5.4.45.37.82.92.82 2.02v3.03c0 .27.1.64.73.55A11 11 0 0012 1.27"
                  />
                </svg>
              </div>
            </div>
            <div
              className="MuiListItemText-root css-tlelie-MuiListItemText-root"
              onClick={[Function]}
            >
              <span
                className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
              >
                Github
              </span>
            </div>
          </div>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
